build/Dockerfile

```diff
@@ -6,36 +6,38 @@ RUN dpkg --add-architecture i386 \
 	&& apt-get update -y \
 	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
 	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install gosu.
```

Comment:
Can't follow you. :) The `RUN` directive represents/creates one layer. The layer gets cached for the build process if it succeeds. If we have two layers, one for `apt-get` and one for `gosu`, we can change the `gosu` layer and the `apt-get` layer will still be cached. If we only have one layer, and we change *anything* in that layer, the whole layer has to be rebuilt. That means, having two layers, is more flexible. I don't see any advantage using one layer, in this case.

Am I missing something?